lib/manageiq/loggers/version.rb

```diff
@@ -1,5 +1,5 @@
 module ManageIQ
   module Loggers
-    VERSION = "0.5.0".freeze
+    VERSION = "0.5.1".freeze
```

Comment:
Please don't change the version in this pull request.  It will be updated at release time.

Reply:
Removed, that commit, sorry about that :+1: